pyaedt/modeler/Model3D.py

```diff
@@ -556,9 +555,6 @@ def import_nastran(self, file_path, import_lines=True, lines_thickness=0):
         -------
         List of :class:`pyaedt.modeler.Object3d.Object3d`
         """
-        if is_ironpython:
-            self._app.logger.error("Method not supported in ironpython. Try import_3d_cad method.")
```

Comment:
The message string should be a complete sentence, such as:

"Method is not supported in IronPython. Try the import_3d_cad method."

Reply:
I've removed and made the method compatible with ironpython.Thanks for suggestion